examples/how_to/create_slave.py

```diff
@@ -1,13 +1,14 @@
 import logging
 from jenkinsapi.jenkins import Jenkins
+from jenkinsapi.utils.crumb_requester import CrumbRequester
 
 log_level = getattr(logging, 'DEBUG')
 logging.basicConfig(level=log_level)
 logger = logging.getLogger()
 
 jenkins_url = "http://localhost:8080/"
-
-api = Jenkins(jenkins_url)
+crumb_requester = CrumbRequester(baseurl=jenkins_url)
```

Comment:
Could you undo changes to this file and make separate one that shows how to work with CSRF enabled Jenkins?

Thanks!